service.py

```
@@ -179,12 +181,12 @@ def get_models(self) -> ModelListResponse:
     @fastapi.get("/models/{model_name:path}")
     def get_model(
         self,
-        model_name=Annotated[str, Path(example="Systran/faster-distil-whisper-large-v2")],
+        model_name=Annotated[str, FastAPIPath(example="Systran/faster-distil-whisper-large-v2")],
```

Ruff (0.15.10)

[warning] Do not perform function call `FastAPIPath` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)